pkg_resources/__init__.py

```diff
@@ -3454,7 +3458,7 @@ def __eq__(self, other: object):
     def __ne__(self, other):
         return not self == other
 
-    def __contains__(self, item: Distribution | str | tuple[str, ...]) -> bool:
+    def __contains__(self, item: Distribution | str) -> bool:
```

Comment:
```python
from packaging import specifiers as _packaging_specifiers
# ...
    def __contains__(
        self, item: Distribution | _packaging_specifiers.UnparsedVersion
    ) -> bool:
```
or
```suggestion
    def __contains__(
        self, item: Distribution | _packaging_version.Version | str
    ) -> bool:```

Reply:
Applied in 0e1e7078560708b450ff5fdd59b9c1f425b15595.